read_test.go

```diff
@@ -410,6 +414,8 @@ func TestReadWithCallback(t *testing.T) {
 		[]string{"sect1", "subsect1", "key5", "", "false"},
 		[]string{"sect1", "subsect2", "", "", "true"},
 		[]string{"sect2", "", "", "", "true"},
+		[]string{"sect3", "", "", "", "true"},
```

Comment:
Do we need this test case?
```suggestion
```

Reply:
I believe so, `ReadWithCallback` will trigger the callback first for the section, then for the key/value. See for example `[]string{"sect1", "", "", "", "true"},` a few lines up.